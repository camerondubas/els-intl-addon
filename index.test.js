const { onComplete } = require('./index');

describe('empty autocomplete', () => {
    it('should not autocomplete if no data', () => {
        expect(onComplete('', {
            results: [],
            type: 'template',
            position: {
                character: 1,
                line: 1
            },
            focusPath: {
                node: {
                    type: 'StringLiteral',
                    value: ''
                },
                parent: {
                    type: 'MustacheStatement',
                    path: {
                        original: 't'
                    }
                }
            }
        })).toMatchSnapshot();
    })
});
