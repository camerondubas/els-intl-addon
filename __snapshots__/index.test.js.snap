// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`empty autocomplete should not autocomplete if no data 1`] = `Array []`;
